File: app/src/main/java/com/omvp/app/ui/samples/requestphone/view/SampleRequestPhoneView.java
package com.omvp.app.ui.samples.requestphone.view;

import com.omvp.app.base.mvp.view.BaseView;

public interface SampleRequestPhoneView extends BaseView {

    void showPrefixInputError(String error);

    void showPhoneInputError(String error);

    void showPrefixInputSuccess();

    void showPhoneInputSuccess();

    void drawInputCode(String code);

    void codeReceivedLayoutVisibility(int visibility);
}

File: app/src/main/java/com/omvp/app/ui/samples/requestphone/view/SampleRequestPhoneView.kt
package com.omvp.app.ui.samples.requestphone.view

import com.omvp.app.base.mvp.view.BaseView

interface SampleRequestPhoneView: BaseView {

    fun showPrefixInputError(error: String)

    fun showPhoneInputError(error: String)

    fun showPrefixInputSuccess()

    fun showPhoneInputSuccess()

    fun drawInputCode(code: String)

    fun codeReceivedLayoutVisibility(visibility: Int)
}
